{
  "Faults": [
    {
      "latitude": 50.70651795841105,
      "type": "LV",
      "reference": "KK0140",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-09T13:55:00.000+0000",
      "affectedCustomerCount": 2,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "5d89f437-2bd3-426d-bf13-ccf5091574c2",
      "minLongitude": -1.2968,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.2935628803159547,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-09T22:15:00.000+0000",
      "estimatedRestoration": "2023-01-10T06:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK0140'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK0140",
      "maxLongitude": -1.2881,
      "minLatitude": 50.7043,
      "name": "PO30 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO30 5UX"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.2968,
              50.7096
            ],
            [
              -1.2881,
              50.7096
            ],
            [
              -1.2881,
              50.7043
            ],
            [
              -1.2968,
              50.7043
            ],
            [
              -1.2968,
              50.7096
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7096,
      "etrUpdate": true
    },
    {
      "latitude": 51.749126372246096,
      "type": "LV",
      "reference": "KK0333",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-09T18:30:00.000+0000",
      "affectedCustomerCount": 12,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "5a413ce8-1fae-439a-880a-e113dfc5d610",
      "minLongitude": -1.2309,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.2292298206312713,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-09T17:34:00.000+0000",
      "estimatedRestoration": "2023-01-10T04:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK0333'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK0333",
      "maxLongitude": -1.2272,
      "minLatitude": 51.7483,
      "name": "OX4 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "OX4 1BT",
        "OX4 1NJ",
        "OX4 1NL"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.2309,
              51.7496
            ],
            [
              -1.2272,
              51.7496
            ],
            [
              -1.2272,
              51.7483
            ],
            [
              -1.2309,
              51.7483
            ],
            [
              -1.2309,
              51.7496
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.7496,
      "etrUpdate": true
    },
    {
      "latitude": 51.429899922740226,
      "type": "LV",
      "reference": "KK0296",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-09T18:15:00.000+0000",
      "affectedCustomerCount": 22,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "18b05817-4bb4-4336-800d-295be1c13c44",
      "minLongitude": -0.9107,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.9077375780091194,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-09T18:15:00.000+0000",
      "estimatedRestoration": "2023-01-10T04:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK0296'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK0296",
      "maxLongitude": -0.9047,
      "minLatitude": 51.4288,
      "name": "RG6 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG6 3AP",
        "RG6 3AR",
        "RG6 3AS",
        "RG6 3AX",
        "RG6 3AY"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.9107,
              51.4318
            ],
            [
              -0.9047,
              51.4318
            ],
            [
              -0.9047,
              51.4288
            ],
            [
              -0.9107,
              51.4288
            ],
            [
              -0.9107,
              51.4318
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4318,
      "etrUpdate": true
    },
    {
      "latitude": 51.377721024003215,
      "type": "LV",
      "reference": "KK0463",
      "etaUpdate": true,
      "affectedCustomerCount": 1,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "94902245-5207-452e-a1e1-be1c7d3e5e80",
      "minLongitude": -1.133,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.1266292713914443,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-09T23:49:00.000+0000",
      "estimatedRestoration": "2023-01-10T03:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK0463'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK0463",
      "maxLongitude": -1.1195,
      "minLatitude": 51.3727,
      "name": "RG7 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG7 4NT",
        "RG7 4NX"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.133,
              51.3823
            ],
            [
              -1.1195,
              51.3823
            ],
            [
              -1.1195,
              51.3727
            ],
            [
              -1.133,
              51.3727
            ],
            [
              -1.133,
              51.3823
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.3823,
      "etrUpdate": true
    },
    {
      "latitude": 51.02816798012883,
      "type": "LV",
      "reference": "KJ8598",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-07T16:05:00.000+0000",
      "affectedCustomerCount": 5,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "eae55afb-58a0-45f2-bf4c-f2197f99b5f0",
      "minLongitude": -0.9458,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.9405200750541067,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-10T01:00:00.000+0000",
      "estimatedRestoration": "2023-01-10T03:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ8598'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ8598",
      "maxLongitude": -0.9355,
      "minLatitude": 51.0257,
      "name": "GU32 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GU32 2DJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.9458,
              51.0323
            ],
            [
              -0.9355,
              51.0323
            ],
            [
              -0.9355,
              51.0257
            ],
            [
              -0.9458,
              51.0257
            ],
            [
              -0.9458,
              51.0323
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.0323,
      "etrUpdate": true
    },
    {
      "latitude": 58.280766392579906,
      "type": "HV",
      "reference": "KK0429",
      "etaUpdate": true,
      "affectedCustomerCount": 74,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "cb691d50-1d71-4011-a48f-38137b946828",
      "minLongitude": -6.6884,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -6.3489633183722045,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-09T21:56:00.000+0000",
      "estimatedRestoration": "2023-01-10T04:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK0429'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK0429",
      "maxLongitude": -6.307,
      "minLatitude": 58.2667,
      "name": "HS2 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "HS2 0JP",
        "HS2 0LP",
        "HS2 0LR",
        "HS2 9BD"
      ],
      "location": {
        "coordinates": [
          [
            [
              -6.6884,
              58.3406
            ],
            [
              -6.307,
              58.3406
            ],
            [
              -6.307,
              58.2667
            ],
            [
              -6.6884,
              58.2667
            ],
            [
              -6.6884,
              58.3406
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-09T22:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 58.3406,
      "etrUpdate": true
    },
    {
      "latitude": 56.50065978945419,
      "type": "LV",
      "reference": "KK0420",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-01-09T22:30:00.000+0000",
      "affectedCustomerCount": 7,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "bec17aca-f18b-47d6-a23b-ae944a5b1195",
      "minLongitude": -2.7619,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.7504141459815106,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-01-09T23:00:00.000+0000",
      "estimatedRestoration": "2023-01-10T01:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KK0420'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KK0420",
      "maxLongitude": -2.7449,
      "minLatitude": 56.4971,
      "name": "DD7 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DD7 7RN",
        "DD7 7RP",
        "DD7 7RU"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.7619,
              56.5036
            ],
            [
              -2.7449,
              56.5036
            ],
            [
              -2.7449,
              56.4971
            ],
            [
              -2.7619,
              56.4971
            ],
            [
              -2.7619,
              56.5036
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-01-09T23:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 56.5036,
      "etrUpdate": true
    }
  ]
}
